libs/sdk-ui-charts/src/highcharts/chartTypes/_chartOptions/chartOptionsBuilder.ts

```diff
@@ -407,8 +407,9 @@ export function getChartOptions(
         ? getCategoriesForTwoAttributes(viewByAttribute, viewByParentAttribute, emptyHeaderTitle)
         : getCategories(type, measureGroup, viewByAttribute, stackByAttribute, emptyHeaderTitle);
 
-    // When custom sorting is enabled and is Pie|Donut chart, need to skip this, so the sort specified by the user does not get override.
-    if ((isPieOrDonutChart(type) && !config.enableChartSorting) || isFunnel(type)) {
+    // When custom sorting is enabled and is Pie|Donut|Funnel chart, need to skip this, so the sort specified by the user does not get override.
+    const isAutoSortableChart = isPieOrDonutChart(type) || isFunnel(type);
```

Comment:
would not be more readble and extensible? 
const autoSortableCharts = ["pie", "donut", "funnel"]
isOneOfTypes(type, autoSortableCharts)